core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/service/impl/MultitableDatasourceService.java

```diff
@@ -58,7 +58,7 @@
   private DatabaseMeta databaseMeta;
   private ConnectionServiceImpl connectionServiceImpl;
   private Log logger = LogFactory.getLog( MultitableDatasourceService.class );
-  private static final String DATASERVICES_CONNECTION = "Local Dataservices Connection";
+  private static final String DATASERVICES_CONNECTION = "Pentaho Data Services";
```

Comment:
Can we replace this static string with new PDIDialect().getDatabaseType.getName()